docker-compose.yml

```diff
@@ -7,13 +7,17 @@ services:
     volumes:
       - './:/srv/pim'
     working_dir: '/srv/pim'
+    networks:
+      - 'akeneo'
 
   node:
-    image: 'node:10-slim'
+    image: 'akeneo/node:10'
```

Comment:
No, this diff is wanted

Reply:
There are no tests on standard, so no need to have a 1.4 GB docker image.

Reply:
But how dev from integrators develops their JS front without using yarn and webpack?

Reply:
Yarn is provided by `node:10-slim`, and webpack is provided by `yarn install`

Reply:
Our custom image adds "only" Chrome and Puppeteer, for tests. Everything needed for building the PIM is already there in the `node:10-slim` image.